src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,113 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createRequest = "INSERT INTO drivers (name, license_number)"
+                + "VALUES (?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createStatement = connection.prepareStatement(createRequest,
+                         Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(1, driver.getName());
+            createStatement.setString(2, driver.getLicenseNumber());
+            createStatement.executeUpdate();
+            ResultSet resultSet = createStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't create driver", throwables);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getRequest = "SELECT * FROM drivers WHERE "
+                + "id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                  PreparedStatement preparedStatement = connection.prepareStatement(getRequest)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            Driver driver = null;
+            if (resultSet.next()) {
+                driver = getDriver(resultSet);
+            }
+            return Optional.ofNullable(driver);
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't get driver from Db",
```

Comment:
Use informative message of exceptions.
      Wrong:
          throw new DataProcessingException("Can't get manufacturer", e);
            
      Good:
          throw new DataProcessingException("Can't get manufacturer by id " + id, e);
          throw new DataProcessingException("Can't insert manufacturer " + manufacturer, e);

Reply:
Okay